types/algoliasearch/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for algoliasearch-client-js 3.30.1
+// Type definitions for algoliasearch-client-js 3.30.8
```

Comment:
this change isn't necessary, it's only "earliest version supporting these typings"